apps/web-giddh/src/app/subscription/view-subscription/view-subscription.component.html

```diff
@@ -119,6 +119,39 @@ <h4 class="font-16">{{ commonLocaleData?.app_view }} {{ localeData?.tab_heading
                             </span>
                         </div>
                         <h4 class="border-bottom pb-3 font-16 mr-t2">{{ commonLocaleData?.app_total }}</h4>
+                        <div class="d-flex justify-content-between pt-3">
+                            <div>
+                                <p class="light-gray mb-1 font-12">Subscription Id</p>
+                                <span class="font-20">
+                                    {{ subscriptionId }}
+                                </span>
+                            </div>
+                            <div *ngIf="viewSubscriptionData?.taxTotal">
+                                <p class="light-gray mb-1 font-12">GST Tax Total</p>
```

Comment:
Translation

Reply:
Nisha will be working on translation and designing